readme.md

```diff
@@ -69,3 +69,11 @@ TODO: map picture
 - interactive visualisations
 	- hide less important lines to reduce noise, show them on zoom
 	- route planning, similar to [vbb-map-routing](https://github.com/derhuerst/vbb-map-routing)
+
+## outlook: interactive modelling tool
+
+We may create an interactive transit map modelling tool.
```

Comment:
We could write something along: "the goal is assist manual editing with an automated optimization approach to find initial solutions or improve existing ones."